docs/pages/build-reference/infrastructure.mdx

```diff
@@ -277,7 +277,25 @@ iOS builder VMs run on Mac mini hosts in an isolated environment. Every build ge
 
 ### iOS server images
 
-#### `macos-sonoma-14.6-xcode-16.0` (`latest`)
+#### `macos-sonoma-14.6-xcode-16.1` (`latest`, `sdk-52`)
+
+<Collapsible summary="Details">
+
+- macOS Sonoma 14.6
+- Xcode 16.1 (16B40)
+- Node.js 18.18.0
+- Bun 1.1.33
+- Yarn 1.22.21
+- pnpm 9.12.3
+- npm 9.8.1
+- fastlane 2.225.0
+- CocoaPods 1.16.0
+- Ruby 3.2
+- node-gyp 10.2.0
+
+</Collapsible>
+
+#### `macos-sonoma-14.6-xcode-16.0`
```

Comment:
Is there supposed to be an SDK version listed here?

Reply:
Do you mean the SDK tag? No, 16.0 is not tagged as any `sdk-xyz` at the moment, but 15.4 is.

Reply:
Yes. Thanks for the clarification!